src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1028,7 +1012,23 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
     ):
-
+        r"""
+        Returns: Example:
```

Comment:
```suggestion
        Returns:
        
        Example:
```
The returns statement needs to be on its own line.